src/main/java/org/broadinstitute/hellbender/engine/dataflow/DataflowCommandLineProgram.java

```diff
@@ -52,7 +52,7 @@ public String getHelpDoc(){
 
     @Argument(doc = "path to the client secrets file for google cloud authentication, necessary if accessing data from buckets.",
             shortName = "secret", fullName = "client_secret", optional=true)
-    protected File clientSecret = new File("client-secrets.json");
+    protected File clientSecret = new File("client_secret.json");
```

Comment:
Add a note in the documentation for this argument that its value will be ignored if the apiKey argument is specified.